hw/ip/otbn/doc/_index.md

```diff
@@ -275,6 +275,16 @@ In addition to the Wide Data Registers, BN instructions can also access WLEN-siz
         A random number.
       </td>
     </tr>
+    <tr>
+      <td>0x3</td>
+      <td>RW</td>
+      <td>
+        <strong>ACC</strong>
+        MAC Accumulator.
+        This gives direct access to the accumulator register utilised by the
+        BN.MULQACC instruction. It is not mapped into the CSRs.
```

Comment:
I'd remove the last sentence; CSRs mapped to WSRs are an exception, not the rule.